app/views/admin/users/_menu.html.erb

```diff
@@ -22,28 +22,15 @@
 </nav>
 
 <% if params[:controller] == "admin/users" %>
-  <nav aria-label="People groups" class="flex justify-between gap-4">
-    <ul class="crayons-navigation crayons-navigation--horizontal">
-      <li>
-        <%= link_to "All", admin_users_path, class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/users' && params[:role].blank?}" %>
-      </li>
-      <li>
-        <%= link_to "Admins", admin_users_path(role: :admin), class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/users' && params[:role] == 'admin'}" %>
-      </li>
-      <li>
-        <%= link_to "Super Admins", admin_users_path(role: :super_admin), class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/users' && params[:role] == 'super_admin'}" %>
-      </li>
-      <li>
-        <%= link_to "Trusted", admin_users_path(role: :trusted), class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/users' && params[:role] == 'trusted'}" %>
-      </li>
-      <li>
-        <%= link_to "Tag Mods", admin_users_path(role: :tag_moderator), class: "crayons-navigation__item #{'crayons-navigation__item--current' if params[:controller] == 'admin/users' && params[:role] == 'tag_moderator'}" %>
-      </li>
-    </ul>
-    <%= form_with url: admin_users_path, method: :get, local: true, class: "flex gap-2" do |f| %>
-      <%= f.text_field :search, value: params[:search], class: "crayons-textfield", placeholder: "Search..." %>
-      <%= f.hidden_field :role, value: params[:role] if params[:role].present? %>
-      <%= f.submit "Search", class: "c-btn c-btn--secondary" %>
-    <% end %>
-  </nav>
+  <%= form_with url: admin_users_path, method: :get, local: true, class: "flex flex-col m:flex-row gap-3 m:items-center crayons-card crayons-card--secondary p-3" do |f| %>
+    <div class="crayons-field flex-1 flex-row items-center gap-2">
+      <%= f.label :search, class: "crayons-field__label whitespace-nowrap" %>
+      <%= f.text_field :search, value: params[:search], class: "crayons-textfield mt-0", placeholder: "Name, username, email, or Twitter/GitHub usernames", aria: { label: "Filter by name, username, or Twitter/GitHub usernames" } %>
```

Comment:
Super small non-blocking question: should we add "email" to the list of available filters for the `aria` label since it is possible to filter by email and since the placeholder contains it in its list?
```suggestion
      <%= f.text_field :search, value: params[:search], class: "crayons-textfield mt-0", placeholder: "Name, username, email, or Twitter/GitHub usernames", aria: { label: "Filter by name, username, email, or Twitter/GitHub usernames" } %>
```

Reply:
Ah yep, nice catch!